### ISSUE TITLE
Broken armor rendering

### ISSUE THREAD

**Issue by Skidamek** [Sinytra/Connector#156]
### Describe the bug

Some armor doesn't render as it does on fabric with mod installed

### Steps to reproduce

1. Install [Show Me Your Skin](https://modrinth.com/mod/show-me-your-skin)
2. Put a leather armor on yourself
3. Change the setting of armor hiding in the mod
![2023-08-31_18 14 41](https://github.com/Sinytra/Connector/assets/67871298/0f763e2f-344d-4c2f-8994-ff9e364e60d6)

**Comment by Skidamek** [comment]
This is from actions latest build btw https://github.com/Sinytra/Connector/actions/runs/6039416430

**Comment by Su5eD** [comment]
Thanks for the report. It should work fine now. I've also tested some modded armor items with it.  
However, a few mods such as [immersive armors](https://www.curseforge.com/minecraft/mc-mods/immersive-armors) use mixins to render their armor, and won't be compatible with transparency. Other mods I've tested - soulslike weaponry, forbidden & arcanus - worked as expected.
![](https://github.com/Sinytra/Connector/assets/51261569/30457920-6122-4a95-ad95-a9095e2481b6)
